### ISSUE TITLE
external endpoint are broken

### ISSUE THREAD

**Issue by eikichi18** [theriverman/django-minio-backend#42]
Hi, 
I've upgrade my project to version 3.4.0 of django_minio_backend and now external URL seems to be broken.
Correct me if I'm wrong but now MINIO_EXTERNAL_ENDPOINT seems to not get used to calculate external url.

Am I right?

**Comment by theriverman** [comment]
Hi,
It is supposed to be used. Go to https://github.com/theriverman/django-minio-backend/blob/master/django_minio_backend/models.py hit Ctrl + F and search for `MINIO_EXTERNAL_ENDPOINT`.

Could you share the relevant lines from your `settings.py` maybe?

**Comment by theriverman** [comment]
sorry for making you wait, @eikichi18 
i could reproduce the issue and i've already found the root cause of it. it has been fixed in 1bd19be.
we've introduced this bug while adding region-awareness to the package in #34.

i've pushed version 3.5.0 to PyPI which now respects the the value of `MINIO_EXTERNAL_ENDPOINT` while generating public urls (private urls were generated correctly already).

@subho007 fyi, this might be impacting you too. consider upgrading to 3.5.0
